testsuite/forge/src/backend/k8s/helm-values/aptos-node-default-values.yaml

```diff
@@ -1,14 +1,16 @@
 validator:
   enableNetworkPolicy: false
-  rust_log: debug,hyper=off
+  # Add debug here to enable debug logs
+  # rust_log: debug,hyper=off
+  rust_log: hyper=off
   # force enable the telemetry service to try to send telemetry
   force_enable_telemetry: true
 
 fullnode:
   # at most one VFN per validator, depending on numFullnodeGroups
   groups:
-  - name: fullnode
-    replicas: 1
+    - name: fullnode
+      replicas: 1
   rust_log: debug,hyper=off
```

Comment:
Should we change this, too? 😄

Reply:
Technically dont need to my formatter did this.... and i believe the yaml result is the same

```
$ yq banana.yaml
banana:
  - this:
    is: a banana

$ yq banana2.yaml
banana:
  - this:
    is: a banana
    
$ diff -u banana.yaml banana2.yaml
--- banana.yaml  2024-04-16 14:23:55
+++ banana2.yaml        2024-04-16 14:24:14
@@ -1,3 +1,3 @@
 banana:
-- this:
-  is: a banana
+  - this:
+    is: a banana
```

Reply:
Oh, no, I meant line 14 has debug logs set for the VFNs? Should we change it to info as well? 😄

Reply:
Good catch ser 🤦